MUDscape/driver.py

```diff
@@ -73,6 +82,22 @@ def send(self, message='', prompt=True):
 
         self.request.sendall(message.format(**Fore.__dict__).encode('utf-8'))
 
+    def bufferInput(self):
+        data = bytes()
+        newLine = "b\'\\r\\n\'"
+        backspace = "b\'\\x08\'"
+        delete = "b\'\\x7f\'"
+        #print(newLine, backspace, delete)
+
+        while True:
+            nextChar = self.request.recv(1024)
```

Comment:
I don't see how this solves the problem of data initially reading 1 character even if it's set to 1024